### PR TITLE
Fix/class attributes

### PR DESCRIPTION
**Pull request by FilipDolnik**

### CODE REVIEW

File: Source/CuckooGeneratorFramework/FileHeaderHandler.swift [hunk 1]
```diff
@@ -18,37 +18,67 @@ public struct FileHeaderHandler {
...
-        let utf8Header = file.sourceFile.contents.utf8.prefix(headerEnd)
-        let header = String(utf8Header) ?? ""
+        let header = getHeader(file)
         return [generationInfo, header]
     }
     
-    public static func getImports(testableFrameworks: [String]) -> [String] {
-        return ["import Cuckoo"] + testableFrameworks.map { "@testable import \($0)" }
+    public static func getImports(file: FileRepresentation, testableFrameworks: [String]) -> [String] {
+        var imports = Array(Set(file.declarations.only(Import).map { "import " + $0.library })).sort()
```
Comment (TadeasKriz) [inline Source/CuckooGeneratorFramework/FileHeaderHandler.swift:26]: Do we need to sort it?

File: Source/CuckooGeneratorFramework/FileHeaderHandler.swift [hunk 2]
```diff
@@ -18,37 +18,67 @@ public struct FileHeaderHandler {
...
+        if imports.count != 0 {
```
Comment (TadeasKriz) [inline Source/CuckooGeneratorFramework/FileHeaderHandler.swift]: `if imports.isEmpty == false`

File: Source/CuckooGeneratorFramework/Tokenizer.swift [hunk 3]
```diff
@@ -82,7 +83,7 @@ public struct Tokenizer {
                 children: children)
             
         case Kinds.ExtensionDeclaration.rawValue:
-            return ExtensionDeclaration()
+            return ExtensionDeclaration(range: range!)
```
Comment (TadeasKriz) [inline Source/CuckooGeneratorFramework/Tokenizer.swift:86]: `!` ?

File: Source/CuckooGeneratorFramework/Tokenizer.swift [hunk 4]
```diff
@@ -205,4 +206,30 @@ public struct Tokenizer {
...
+                return nil
+            }
+        }
+        do {
+            let regex = try NSRegularExpression(pattern: "(?:\\b|;)import(?:\\s|(?:\\/\\/.*\\n)|(?:\\/\\*.*\\*\\/))+([^\\s;\\/]+)", options: [])
+            let results = regex.matchesInString(source, options: [], range: NSMakeRange(0, source.characters.count))
+            return results.filter { result in
+                    rangesToIgnore.filter { $0 ~= result.range.location }.count == 0
+                }.map {
+                    let range = $0.range.toRange()!
```
Comment (TadeasKriz) [inline Source/CuckooGeneratorFramework/Tokenizer.swift:228]: `!` ?

File: Source/CuckooGeneratorFramework/Tokenizer.swift (hunk 3 above)
Comment (FilipDolnik) [inline Source/CuckooGeneratorFramework/Tokenizer.swift:86]: This range is everywhere used with !.

File: Source/CuckooGeneratorFramework/Tokenizer.swift (hunk 4 above)
Comment (FilipDolnik) [inline Source/CuckooGeneratorFramework/Tokenizer.swift:228]: I can change it to `$0.range.location..<($0.range.location + $0.range.length)`, which should produce the same result.

File: Source/CuckooGeneratorFramework/FileHeaderHandler.swift (hunk 1 above)
Comment (FilipDolnik) [inline Source/CuckooGeneratorFramework/FileHeaderHandler.swift:26]: It is for nicer output. Also it is predictable which is useful in tests.

**Pull request merged**